test/dynamo/test_export.py

```diff
@@ -320,6 +320,7 @@ def func(x, z, k):
 
         self.assertTrue(torch._dynamo.utils.same(real_result, dynamo_result))
 
+    @patch.object(torch._dynamo.config, "dynamic_shapes", True)
```

Comment:
this is already run under test/dynamo/test_dynamic_shapes.py, unless you want this to only run under dynamic shapes?

Reply:
Saw rest of pr, sg

Reply:
There is a problem which is when we mark things as dynamic shapes True here, we actually are technically running the test twice but under same config. This is annoying to fix though, and is preexisting condition